addons/website_payment/static/src/js/website_payment_form.js

```diff
@@ -53,7 +53,6 @@ checkoutForm.include({
                     const $field = this.$('input[name="' + id + '"],select[name="' + id + '"]');
                     $field.addClass('is-invalid');
                     $field.popover({content: errorFields[id], trigger: 'hover', container: 'body', placement: 'top'});
-                    $field.data("bs.popover").config.content = errorFields[id];
```

Comment:
Bootstrap 5 equivalent code would be:
```js
const popover = Popover.getInstance($field);
popover._config.content = errorFields[id];
popover.setContent(popover.getTipElement());
```
But doesn't seem needed at all.

@bso-odoo Any chance you remember why you wrote it in the [first place](https://github.com/odoo/odoo/commit/7fccbac004628093da49016f75370a84bba49465#diff-197f0de1c9f51c575093e9c41cc5f6c8c8afcc75a20c3093cd4a234c18c82b59R44) ? Maybe I am missing something obvious here.

Reply:
I cannot find an explanation about it: it even works in 15.0 without it. From what I read online, it is used as a trick to avoid losing the position of the popover when replacing the content. I doubt this is needed here for that reason given that we initialize the popover with its text and position at the previous line.
I found two other uses of that mechanism at the time, but they have been adapted for 16.0 in the same way you describe: https://github.com/odoo/odoo/commit/37546006940f99c8860e89997ed7a623abd5fa72 & https://github.com/odoo/odoo/commit/0cff1dc2967cafeb8964ed0802c309d3bb7f7525.
Maybe @bvr-odoo remembers something about it.

Reply:
No, sorry, I have no idea.